halotools/mock_observables/pairwise_velocities/los_pvd_vs_rp.py

```diff
@@ -21,57 +21,60 @@
 
 def los_pvd_vs_rp(sample1, velocities1, rp_bins, pi_max, sample2=None,
         velocities2=None, period=None, do_auto=True, do_cross=True,
-        num_threads=1,
-        approx_cell1_size=None, approx_cell2_size=None, seed=None):
+        num_threads=1, approx_cell1_size=None, approx_cell2_size=None):
     r"""
-    Calculate the pairwise line-of-sight (LOS) velocity dispersion (PVD), :math:`\sigma_{z12}(r_p)`.
+    Calculate the pairwise line-of-sight (LOS) velocity dispersion (PVD),
+    as a function of radial distance from ``sample1`` :math:`\sigma_{z12}(r_p)`.
 
     Example calls to this function appear in the documentation below.
 
     Parameters
     ----------
     sample1 : array_like
-        Npts x 3 numpy array containing 3-D positions of points.
+        Npts1 x 3 numpy array containing 3-D positions of points.
 
     velocities1 : array_like
-        N1pts x 3 array containing the 3-D components of the velocities.
+        Npts1 x 3 array containing the 3-D components of the velocities.
 
     rp_bins : array_like
         array of boundaries defining the radial bins perpendicular to the LOS in which
         pairs are counted.
+        Length units are comoving and assumed to be in Mpc/h, here and throughout Halotools.
 
     pi_max : float
         maximum LOS separation
+        Length units are comoving and assumed to be in Mpc/h, here and throughout Halotools.
 
     sample2 : array_like, optional
-        Npts x 3 array containing 3-D positions of points.
+        Npts2 x 3 array containing 3-D positions of points.
 
     velocities2 : array_like, optional
-        N1pts x 3 array containing the 3-D components of the velocities.
+        Npts2 x 3 array containing the 3-D components of the velocities.
 
     period : array_like, optional
         length 3 array defining  periodic boundary conditions. If only
         one number, Lbox, is specified, period is assumed to be [Lbox, Lbox, Lbox].
 
     do_auto : boolean, optional
-        caclulate the auto-pairwise velocities?
+        calculate the auto-pairwise velocities?
 
     do_cross : boolean, optional
-        caclulate the cross-pairwise velocities?
+        calculate the cross-pairwise velocities?
 
     num_threads : int, optional
         number of threads to use in calculation. Default is 1. A string 'max' may be used
         to indicate that the pair counters should use all available cores on the machine.
 
-    seed : int, optional
```

Comment:
Like verbose, there are a couple of functions that take a seed but never use it.